spec/requests/portfolio_items_spec.rb

```diff
@@ -68,9 +68,9 @@
   end
 
   describe "POST /portfolios/:portfolio_id/portfolio_items" do
-    let(:params) { {:portfolio_item_id => portfolio_item.id} }
+    let(:params) { {:portfolio_item_id => portfolio_item_id} }
     before do
-      post "#{api}/portfolios/#{portfolio.id}/portfolio_items", :params => params, :headers => default_headers
+      post "#{api}/portfolios/#{portfolio.id}/portfolio_items", :params => params, :headers => default_headers, :as => :json
```

Comment:
If we merge https://github.com/ManageIQ/catalog-api/pull/369 can we remove these `:as => 'json'` arguments?

Reply:
@syncrou we can do that in a separate PR. @Fryguy was thinking that maybe the #369 should be in the manager-api-common gem.